refresh-data.sh

```diff
@@ -43,13 +49,13 @@ while true; do
   if [ "$HAS_BUNDLE_UPDATED" = "0" ]; then
     # Make sure the right versions of dependencies are installed for this
     # repository
-    bundle install
+    bundle install >/dev/null 2>&1
```

Comment:
Just a drive-by comment to say that `bundle install` (and `bundle update`) both have a `--quiet` option to suppress output.

Reply:
Awesome; I'll use that instead :-)